src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-daemon-list/rgw-daemon-list.component.html

```diff
@@ -36,6 +37,7 @@
        i18n>Sync Performance</a>
     <ng-template ngbNavContent>
       <cd-grafana [grafanaPath]="'radosgw-sync-overview?'"
+                  [type]="'metrics'"
```

Comment:
If the changes in my below comments are doable, then we don't need these changes right?

Reply:
To keep the unformity across the code, I decided to go with this, as in whenever the cd-grafana component is called, type should be provided as the input.

Reply:
Okay.